dev_guide/daemonsets.adoc

```diff
@@ -0,0 +1,108 @@
+[[dev-guide-daemonsets]]
+= Using Daemonsets
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+Daemonsets are used to run a copy of a pod on specific or all nodes in an
+{product-title} environment upon node creation.
+
+Use daemonsets to create shared storage, run a logging pod on every node in
+your cluster, or deploy a monitoring agent on every node.
+
+For more information on daemonsets, see the link:http://kubernetes.io/docs/admin/daemons/[Kubernetes documentation].
+
+[[dev-guide-creating-daemonsets]]
+== Creating Daemonsets
+
+[IMPORTANT]
+====
+Before creating daemonsets, ensure you have been
+xref:../admin_guide/manage_authorization_policy.adoc#admin-guide-granting-users-daemonset-permissions[given
+the required role by your {product-title} administrator].
+====
+
+When creating Daemonsets, the `*nodeSelector*` field is used to indicate the
+nodes on which the Daemonset should deploy onto.
+
+. Define the daemonset yaml file:
```

Comment:
Do we need to create this for 3.3, or will this role/profile be available
in 3.3

On Sep 6, 2016 11:48 AM, "Abhishek Gupta" <notifications@github.com> wrote:

> In dev_guide/daemonsets.adoc
> <https://github.com/openshift/openshift-docs/pull/2786#discussion_r77662182>
> :
>
> > +For more information on daemonsets, see the link:http://kubernetes.io/docs/admin/daemons/[Kubernetes documentation].
> > +
> > +[[dev-guide-creating-daemonsets]]
> > +== Creating Daemonsets
> > +
> > +[IMPORTANT]
> > +====
> > +Before creating daemonsets, ensure you have been
> > +xref:../admin_guide/manage_authorization_policy.adoc#admin-guide-granting-users-daemonset-permissions[given
> > +the required role by your {product-title} administrator].
> > +====
> > +
> > +When creating Daemonsets, the `*nodeSelector*` field is used to indicate the
> > +nodes on which the Daemonset should deploy onto.
> > +
> > +. Define the daemonset yaml file:
>
> I am not even sure what the question is - perhaps I am just not aware of
> what is going into 3.3 that will remove the need for creating a daemonset
> file.
> @sferich888 <https://github.com/sferich888> can you please clarify?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/openshift-docs/pull/2786/files/363329d2b6d4802887ff0e62d78e0d7bb5f0444c#r77662182>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABUZmhPA_RvI1JGKkYJBzjy1cO63XlrEks5qnYsZgaJpZM4J0ufO>
> .
>